demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2920,7 +2922,16 @@ export class DemoMeetingApp
       }
     }
 
+    if (value === 'No Audio') {
+      // An empty media stream destination without any source connected to it, so it doesn't generate any audio
+      // This is currently only used for integration testing of 'sendingAudioFailed' and 'sendingAudioRecovered' events.
+      // Note: This only works in Chromium based browsers and doesn't work in Firefox because Firefox generates
```

Comment:
So should we attach the 'No Audio' option only for browsers other than Firefox? If we run into any error/fatal error then probably we should do that. What do you think?

Reply:
It doesn't throw an error/fatal per se, rather it just doesn't work as intended. I do think it's better to hide it in FF just for the sake of avoiding any confusion.